### DISCUSSION TITLE
rAudio 20230528: "Wrong scontrol identifier: "

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1327] [Bug Reports]
The update log to 20230528 ended with the message "Wrong scontrol identifier:"

**Comment by Sagnasty** [comment]
I did not get this message - here is my 20230528 update log if it helps
[rAudio 20230528 update log.txt](https://github.com/rern/rAudio/files/11584901/rAudio.20230528.update.log.txt)

**Reply by klassikmann** [reply]
> I did not get this message - here is my 20230528 update log if it helps [rAudio 20230528 update log.txt](https://github.com/rern/rAudio/files/11584901/rAudio.20230528.update.log.txt)

I assume the origin of my additional message is a feature of rAudio I didn't select.

**Comment by klassikmann** [comment]
Still present in 20230609 (but a minor inconvenience only).

**Comment by klassikmann** [comment]
Fixed in rAudio 20230615

**Comment by klassikmann** [comment]
**Re-opened:** back again in rAudio 20230630
